app/components/polls/polls-edit/polls-edit.html

```diff
@@ -1,143 +1,195 @@
-<top-bar
-    logo-href="/cms/app/polls/"
-    save-function="saveModel"
-    save-disable-when="model.$dirty().length === 0 && !isNew">
-</top-bar>
+<ng-form name="pollForm">
+  <top-bar
+      logo-href="/cms/app/polls/"
+      save-function="saveModel"
+      save-disable-when="(model.$dirty().length === 0 && !isNew) || !pollForm.$valid">
```

Comment:
Qucik check: it looks like that isn't a straight-up replacement so I'd like to consider it out of scope for this PR.